### PR TITLE
feat: add node metrics doc

### PR DESCRIPTION
**Pull request by Juuddi**
- Add node metrics config doc as per: https://github.com/dominant-strategies/go-quai/pull/1704/files
- Rework a few of the sidebar_labels for node, miner, stratum

### CODE REVIEW

**Review by gameofpointers — CHANGES_REQUESTED** [review]
why are we commiting a random package-lock.json ? @Juuddi

**Comment by Juuddi** [comment]
> why are we commiting a random package-lock.json ? @Juuddi

Cleaned out the old deps and reinstalled, should just be the latest package-lock

File: static/img/GrafanaDashboard.png
Comment (Djadih) [inline static/img/GrafanaDashboard.png:1]: Is there space to include the other screenshots? We should actually have explainers on what each metric means to the average user

**Review by Djadih — CHANGES_REQUESTED** [review]

File: static/img/GrafanaDashboard.png
Comment (Juuddi) [reply static/img/GrafanaDashboard.png:1]: Yeah there's plenty of room, just drop me some explainers on all of the metrics you want to highlight and I can throw them in

Comment (Djadih) [reply static/img/GrafanaDashboard.png:1]: Should be similar to this
https://geth.ethereum.org/docs/monitoring/understanding-dashboards

Comment (Djadih) [reply static/img/GrafanaDashboard.png:1]: Happy to help if any are unclear just come by my desk, but basically these should be from a user perspective like in the example above

Comment (Juuddi) [reply static/img/GrafanaDashboard.png:1]: Definitely a nice-to-have but for the most part I would assume this content would go in a go-quai specific docs page, similar to how this is in the geth docs, not the ethereum docs.

If you want all of this content in the docs here, I'll need a general overview of what changes between geth and go-quai metrics, to what extent of the content you want to include, and all of the screenshots as well. Might be best for you to write this as you have the subject matter knowledge.

Comment (Djadih) [reply static/img/GrafanaDashboard.png:1]: I see what you mean now, I agree with the distinction between protocol and client docs so if we set that up and have a docs page for go-quai I'm all for it, and that can be done at a future date.

For the second part I don't think that all client related docs require full time effort from protocol development as the user facing changes in this (and many) cases are minimal and imo shouldn't be beyond the expertise of people we have writing the docs (they have experience with the product and direct access to the devs to ask). Considering that product development is still in full swing I just don't think we have the bandwidth or resources to devote protocol resources in every case.

File: docs/participate/node/node-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+  - grafana
+  - prometheus
+  - metrics
+---
+
+# Node Monitoring
+
+## Introduction
+
+Monitoring your node's performance in real time is crucial to ensure that it is running smoothly and efficiently. The go-quai client bundles a monitoring solution that allows you to track the health and performance of your node. This guide will walk you through setting up Grafana and Prometheus to monitor your node.
```
Comment (wizeguyy) [inline docs/participate/node/node-monitoring.md:18]: I might dial this back a bit. 99% of people never need to run metrics or graphana or anything, so I wouldn't call this a crucial activity. Its more of a tool for advanced users who would like to understand the node's performance on their system.

File: docs/participate/stratum-proxy/stratum-faq.md [hunk 2]
```diff
@@ -47,11 +48,11 @@ To properly run this command, you'll need to replace `REGION-WS-PORT` and `ZONE-
...
-| Chain Name | Type   | Chain Index | Web Socket Port | Stratum Command                                         |
-| ---------- | ------ | ----------- | --------------- | ------------------------------------------------------- |
-| Cyprus     | Region |             | 8579            |                                                         |
-| Paxos      | Region |             | 8581            |                                                         |
-| Hydra      | Region |             | 8583            |                                                         |
+| Chain Name | Type   | Chain Index | Web Socket Port | Stratum Command                                            |
+| ---------- | ------ | ----------- | --------------- | ---------------------------------------------------------- |
+| Cyprus     | Region |             | 8579            |                                                            |
+| Paxos      | Region |             | 8581            |                                                            |
+| Hydra      | Region |             | 8583            |                                                            |
```
Comment (wizeguyy) [inline docs/participate/stratum-proxy/stratum-faq.md:55]: :muscle:

File: docs/participate/node/node-monitoring.md (hunk 1 above)
Comment (Juuddi) [reply docs/participate/node/node-monitoring.md]: Agree with you here. Language has been fixed in [29af5bc](https://github.com/dominant-strategies/quai-docs/pull/108/commits/29af5bcdadde6a376392f31ab2806ec9b5ce82ac)

**Review by Djadih — APPROVED** [review]

**Pull request merged**